README.md

```diff
@@ -508,7 +508,7 @@ source:
     map:
       name: ip
       model: model
-      username: username
+      user: username
```

Comment:
@crami the `name` change should be the one at line `506`. could you fix this, please?

ref.: [lib/oxidized/source/sql.rb](https://github.com/ytti/oxidized/blob/c8ca088c1314bc55284fd6bc110a2372d22b1ce4/lib/oxidized/source/sql.rb#L53-L57)